en/flight_controller/cuav_v5_plus.md

```diff
@@ -109,6 +109,47 @@ It does not have an i/o debug interface.
 Any multicopter / airplane / rover or boat that can be controlled with normal RC servos or Futaba S-Bus servos.
 The complete set of supported configurations can be seen in the [Airframes Reference](../airframes/airframe_reference.md).
 
+## Known Issues
+
+#### GPS not compatible with other devices (Critical)
+
+The *Neo v2.0 GPS* recommended for use with *CUAV V5+* and *CUAV V5 nano* is not fully compatible with other Pixhawk flight controllers (specifically, the buzzer part is not compatible and there may be issues with the safety switch).
+The GPS will not work with other flight controllers, and is the only GPS unit that can be used with the *CUAV V5+* and *CUAV V5 nano*.
+<!-- 5+/90/V5+ 20190523 RC01 -->
+
+- *Found:* Batch 01
+- *Fixed:* -
+
+#### Volt regulation varies greater than +/- 5%
+
+The 5 volt pins on all connectors will be lower when powered from USB than the Power Module (the pins will measure approximately 4.69V when only powered by USB, and 5.28 Volts when connected to the Power Module).
+
+> **Note** We recommend that when using USB with the *v5 Nano* you *also connect the power module* (to avoid under-powering any connected peripherals.
+
+- *Found:* Batch 01
+- *Fixed:* -
+
+#### Do not plug Digital or Analog PM onto connectors configured for other type of PM
+
+If you plug an Analog PM into a digital PM connector it will stop all the I2C devices on that bus.
+Specifically this will stop the GPS's compass due to contention, and may also damage the FMU (longer term).
+
+Similarly, a digital PM plugged into a analog connector will not work, and may also damage/destroy the power module (longer term).
+
+- *Found:* Batch 01
+- *Fixed:* -
+
+
+#### Debugging on hardware (developer note)
```

Comment:
@davids5 Note, title slightly more expressive  - because this isn't going to be relevant to all developers. Note also that this isn't an issue as such, so I haven't put a "Found/Fixed" section at the bottom.  HOpe that OK.

Reply:
Not really, one can use printf debugging on HW.  The issue is: (and it is an issue) Using JTAG on V5+ and Nano.

Reply:
See comment in PR

Reply:
OK, that is a better title then. On both topic I changed the heading to "Using JTAG for hardware debugging"